src/Workspaces/Core/Portable/Workspaces.csproj

```diff
@@ -23,6 +23,10 @@
       <Project>{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}</Project>
       <Name>CodeAnalysis</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\CSharpCodeAnalysis.csproj">
```

Comment:
Workspaes should not depend on CSHarpCodeAnalysis.

Reply:
The real answer is not that it _shouldn't_ depend on CSharpCodeAnalysis. Instead, Workspaces _can't_ depend on CSharpCodeAnalysis. This is a layering violation. Language-agnostic projects (like Workspaces) do not get to depend on Language-specific projects.